src/custom_nodes/image_transformation/image_transformation.cpp

```diff
@@ -97,18 +119,29 @@ int execute(const struct CustomNodeTensor* inputs, int inputsCount, struct Custo
         NODE_ASSERT(originalImageColorChannels == 3, "for color order BGR/RGB color channels must be equal to 3");
     }
 
-    targetImageHeight = targetImageHeight == -1 ? originalImageHeight : targetImageHeight;
-    targetImageWidth = targetImageWidth == -1 ? originalImageWidth : targetImageWidth;
+    uint64_t targetImageHeight = _targetImageHeight == -1 ? originalImageHeight : _targetImageHeight;
```

Comment:
Minor:
I think in this form this would be more readable
uint64_t targetImageHeight = _targetImageHeight != -1 ?  _targetImageHeight : originalImageHeight;

Reply:
This is similar convention to other parameters default setting